js/background.js

```diff
@@ -1,80 +1,76 @@
 /* This script runs in the background from when the app is loaded */
 
-var Domain = "https://channeli.in";
-var Host = "channeli.in";
+var NET_CHECK_DOMAIN = "http://info.cern.ch/";
```

Comment:
Use Channeli's static files serving Nginix endpoint here(You can ask Kartik about the details) instead of some external endpoint, to make this extension to be consistent with channeli.